PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -86,6 +86,8 @@ public static Question[] LoadQuestions(string filePath)
             question.Answers[1] = answer2;
             question.Answers[2] = answer3;
             question.CorrectAnswerIndex = correctAnswerIndex;
+            questions[1] = question;
```

Comment:
```suggestion
            questions[i] = question;
```

As Joshua mentioned, this line is causing duplicate output. Instead of calling index `1`, you want the specify the index as `i`.